.env.example

```diff
@@ -1,33 +1,36 @@
-UVICORN_HOST = "0.0.0.0"
-UVICORN_PORT = 8000
+UVICORN_HOST="0.0.0.0"
+UVICORN_PORT=8000
 
 
 ## We highly recommend add admin using `marzban cli` tool and do not use
 ## the following variables which is somehow hard codded infrmation.
-# SUDO_USERNAME = "admin"
-# SUDO_PASSWORD = "admin"
+# SUDO_USERNAME="admin"
+# SUDO_PASSWORD="admin"
 
 # UVICORN_UDS: "/run/marzban.socket"
-# UVICORN_SSL_CERTFILE = "/var/lib/marzban/certs/example.com/fullchain.pem"
-# UVICORN_SSL_KEYFILE = "/var/lib/marzban/certs/example.com/key.pem"
+# UVICORN_SSL_CERTFILE="/var/lib/marzban/certs/example.com/fullchain.pem"
+# UVICORN_SSL_KEYFILE="/var/lib/marzban/certs/example.com/key.pem"
 
+# To change default dashboard path. Don't forget / at the start and the end of path.
+DASHBOARD_PATH=dashboard
```

Comment:
This should be a comment in example file, also you forgot to use / in start and end and " for string

Reply:
/ at start and end will set automatically and ok I add " for string and make this env as a comment